cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-database-syntax.asciidoc

```diff
@@ -16,10 +16,10 @@ GRANT {START \| STOP}
 | Grant the specified roles the privilege to start and stop the default database, specific database(s), or all databases.
 
 | [source, cypher]
-GRANT {CREATE \| DROP} INDEX[ES]
+GRANT {CREATE \| DROP \| SHOW} INDEX[ES]
```

Comment:
I found a list just above where this is rendered (first in the 5.5.6.2. Database administration section) where SHOW INDEX and SHOW CONSTRAINT privileges should also be added